source/extensions/common/wasm/wasm.cc

```diff
@@ -504,6 +504,21 @@ void replaceHeader(Http::HeaderMap* map, absl::string_view key, absl::string_vie
     map->addCopy(lower_key, std::string(value));
 }
 
+const uint8_t* decodeVarint(const uint8_t* start, const uint8_t* end, uint32_t* out) {
```

Comment:
Could you make this function at this moment?

Reply:
Could you clarify?  Do you think the function should be moved?

Reply:
Sorry... The word "static" is eaten by mistake -_-b

I think we could declare the function static so that no other component should use `decodeVarint`
If there is further usage, it indicates we should extract this function to a util class.

Reply:
it is in an anonymous namespace.